packages/survey-creator-knockout/src/components/page.html

```diff
@@ -1,15 +1,13 @@
-<div class="svc-page__content"
-  data-bind="click: select, key2click, clickBubble: false, css: css, attr: { id: page.id }, event: { mouseover: function(m, e) { hover(e, $element); }, mouseleave: function(m, e) { hover(e, $element); } }">
+<div class="svc-page__content" data-bind="click: select, key2click, clickBubble: false, css: css, attr: { id: page.id }, event: { mouseover: function(m, e) { hover(e, $element); }, mouseleave: function(m, e) { hover(e, $element); } }">
   <!-- ko template: { name: 'survey-page', data: page } -->
   <!-- /ko -->
 
   <!-- ko if: allowEdit -->
   <div class="svc-page__add-new-question" data-bind="click: addNewQuestion, key2click, clickBubble: false">
-    <span class="svc-text svc-text--normal svc-text--bold" data-bind="text: creator.addNewQuestionText">
+    <span class="svc-text svc-text--normal svc-text--bold" data-bind="text: $data.addNewQuestionText">
     </span>
     <!-- ko with: questionTypeSelectorModel -->
-    <button data-bind="click: action, key2click, clickBubble: false, attr: { title: creator.addNewQuestionText }"
-      class="svc-page__question-type-selector">
+    <button data-bind="click: action, key2click, clickBubble: false, attr: { title: $data.addNewQuestionText }" class="svc-page__question-type-selector">
```

Comment:
any button should have the `type="button"` attribute otherwise by default it will submit users form